#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
uint8 front_act_pos
uint8 rear_act_pos
uint8 eps_sys_state
uint8 steer_wheel_position
uint8 front_mov_state
uint8 rear_mov_state
uint16 eps_error